internal/admin/siginfo/form.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, softwar
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package siginfo
+
+import (
+	"time"
+
+	"github.com/google/exposure-notifications-server/internal/export/model"
+)
+
+type formData struct {
+	SigningKey        string `form:"SigningKey"`
```

Comment:
Where are these annotations used?

Reply:
Used by Gin in the form binding process

https://github.com/gin-gonic/gin#model-binding-and-validation